Zend/tests/enum/default-parameter.phpt

```diff
@@ -0,0 +1,18 @@
+--TEST--
+Enum in default parameter
+--FILE--
+<?php
+
+enum Foo {
+    case Bar;
+}
+
+function baz(Foo $foo = Foo::Bar) {
+    var_dump($foo);
+}
+
+baz();
```

Comment:
Also, there should be a test of using enums as the default of a static variable in case that's somehow unexpected by opcache or the jit

```
function example() {
    static $baz = Foo::BAR;
    return $baz;
}
var_export(example());
```